tools/convert/ruleconvert.go

```diff
@@ -37,17 +37,20 @@ func transformSamplerMap(m map[string]any) map[string]any {
 		switch k {
 		case "clearfrequencysec":
 			k = "clearfrequency"
-			v = config.Duration(v.(int64)) * config.Duration(time.Second)
+			switch vi := v.(type) {
+			case int:
```

Comment:
nit - I think we can do something like this. I don't know if there's a chance for this value to be types outside of int/int64. Should we add a default case to handle that?
```
case int, int32, int64:
   v = config.Duration(vi) * config.Duration(time.Second)
```

Reply:
I really like this suggestion, much easier to read.

Reply:
That's good to know! Thanks for explaining it